cartridge/failover.lua

```diff
@@ -422,26 +451,10 @@ local function failover_loop(args)
             goto continue
         end
 
-        -- Cancel all pending tasks
-        for id, task in pairs(vars.schedule) do
-            if task:status() == 'dead' then
-                vars.schedule[id] = nil
-            elseif not task.storage.is_busy then
-                vars.schedule[id] = nil
-                task:cancel()
-            -- else
-                -- preserve busy tasks
-            end
-        end
-
-        -- Schedule new task
-        do
-            local task = fiber.new(reconfigure_all, vars.cache.active_leaders)
-            local id = task:id()
-            task:name('cartridge.failover.task')
-            vars.schedule[id] = task
-            log.info('Failover triggered, reapply scheduled (fiber %d)', id)
-        end
+        log.info(
+            'Failover triggered, reapply scheduled (fiber %d)',
+            schedule_add(reconfigure_all, vars.cache.active_leaders)
```

Comment:
I don't like this idea. log function argument performs some complex logic inside. Let's declare a local variable, save result into it and then log it.

Reply:
Fixed